File: data/src/main/java/com/thinkzi/weather/data/model/Sys.java
package com.thinkzi.weather.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Sys {

    @SerializedName("pod")
    @Expose
    private String pod;

    public Sys() {
        this.pod = "";
    }

    public String getPod() {
        return pod;
    }

    public void setPod(String pod) {
        this.pod = pod;
    }

}

File: data/src/main/java/com/thinkzi/weather/data/model/Sys.kt
package com.thinkzi.weather.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Sys {

    @SerializedName("pod")
    @Expose
    var pod: String = ""

    init {
        this.pod = ""
    }

}
